.github/workflows/ci.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     - uses: actions/checkout@v3
     - uses: actions/setup-go@v3
       with:
-        go-version: '1.18'
+        go-version: '1.24'
```

Comment:
The `go.mod` file requires Go 1.23 :thinking: 
Maybe it makes sense to automatically determine the Go version to install from the file:
https://github.com/actions/setup-go?tab=readme-ov-file#getting-go-version-from-the-gomod-file

Reply:
Thanks for the suggestion!  That's good to know.

I tried it and it works but it's weird because the logs indicate it installing both 1.23.1 and 1.24.0, and `go env` ends up with version 1.24.0.